gsa/public/locales/gsa-de.json

```diff
@@ -690,6 +704,7 @@
   "Fold": "Einklappen",
   "Fold all details": "Alle Details einklappen",
   "Follow Link": "Link folgen",
+  "Foo": "Foo",
```

Comment:
???

Reply:
This was more of a joke. It's used in one of our tests and shows up for translation ;)

Reply:
Ah ok. We should not extract strings from tests in future :-)